include/libtorrent/create_torrent.hpp

```diff
@@ -366,8 +366,7 @@ namespace libtorrent {
 		bool m_include_symlinks:1;
 	};
 
-namespace detail {
-
+	namespace detail {
```

Comment:
well, I would prefer no more than one indentation level as the end goal (and probably just make namespaces not indent at all). Small corrections in that direction are fine, I'm procrastinating a bit about changing all the indentations in all files. I don't think it's urgent and it comes with some (merge) risk, so I don't know about that the best approach is there.

in this case, if it's just the ``nop`` function, please fix its indentation